test/e2e/framework/t.go

```diff
@@ -98,5 +100,24 @@ func (t *T) Wait() {
 	}
 }
 
+func (t *T) flush() {
+	t.T.Helper()
+loop:
+	for {
+		select {
+		case err := <-t.errors:
+			t.T.Helper()
```

Comment:
Does this do something in addition to the Helper call on L104?

Reply:
Cargo culting the old code.

Reply:
fixed